jenkins-scripts/dsl/core.dsl

```diff
@@ -34,7 +34,7 @@ update_vcpkg_snapshot_job.with
 def ignition_testing_software = 'gazebo'
 def testing_vcpkg_job = job("_vcpkg_testing_snapshot")
 OSRFWinCompilationAnyGitHub.create(testing_vcpkg_job,
-                                  "ignitionrobotics/ign-${ignition_testing_software}",
+                                  "gazebosim/ign-${ignition_testing_software}",
```

Comment:
```suggestion
                                  "gazebosim/gz-${ignition_testing_software}",
```

Reply:
The rename of the repository was not done on porpoises since it has code implications, particularly since the checkout directory is going to change ([see DSL code](https://github.com/gazebo-tooling/release-tools/blob/master/jenkins-scripts/dsl/_configs_/GenericAnyJobGitHub.groovy#L27)) and we need to test that carefully.

Reply:
additionally I think the `ign-gazebo` to `gz-sim` change will take some extra care and testing